src/boot/efi/log.c

```diff
@@ -81,6 +81,12 @@ void __stack_chk_fail_local(void) {
         __stack_chk_fail();
 }
 
+/* Called by libgcc for some fatal errors like integer overflow with -ftrapv. */
+_used_ _noreturn_ void abort(void);
+void abort(void) {
```

Comment:
Why `_used_` is sometimes set to prototype but sometimes to function?

Reply:
To annoy people?